components/board/pi/impl/board.go

```diff
@@ -215,10 +215,10 @@ func (pi *piPigpio) reconfigureSpis(ctx context.Context, cfg *genericlinux.Confi
 	return nil
 }
 
-func (pi *piPigpio) reconfigureAnalogs(ctx context.Context, cfg *genericlinux.Config) error {
+func (pi *piPigpio) reconfigureAnalogReaders(ctx context.Context, cfg *genericlinux.Config) error {
 	// No need to reconfigure the old analog readers; just throw them out and make new ones.
-	pi.analogs = map[string]board.AnalogReader{}
-	for _, ac := range cfg.Analogs {
+	pi.analogReaders = map[string]board.AnalogReader{}
+	for _, ac := range cfg.AnalogReaders {
```

Comment:
Where is the AnalogReaders config parameter in raspberry pi? Should it change to a `board.MCP3008` object?